pages/product.html

```diff
@@ -0,0 +1,429 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../dist/output.css" />
+    <link rel="stylesheet" href="../index.css" />
+    <title>Product</title>
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <div class="nav-bar__logo">{finsweet</div>
+      <ul class="nav-bar__list">
+        <li class="nav-bar__item">
+          <a href="./index.html">Home</a>
+        </li>
+        <li class="nav-bar__item">
+          <a href="./pages/product.html">Product</a>
+        </li>
+        <li class="nav-bar__item">
+          <a href="./pages/pricing.html">Pricing</a>
+        </li>
+        <li class="nav-bar__item">
+          <a href="./pages/about.html">About Us</a>
+        </li>
+        <li class="nav-bar__item">
+          <a href="./pages/blog.html">Blog</a>
+        </li>
+        <li class="nav-bar__item">
+          <a href="./pages/contact.html">Contact</a>
+        </li>
+        <li class="primary-btn">Free trial</li>
```

Comment:
## prohibition of nested block
`primary-btn` block inside block is not a good practice